Xamarin.Essentials/FilePicker/FilePicker.android.cs

```diff
@@ -67,17 +65,9 @@ internal FilePickerResult(global::Android.Net.Uri contentUri)
 
         readonly global::Android.Net.Uri contentUri;
 
+        // Basically with android 10, its highly discouraged to get the path. Work with the URI.
         static string GetFullPath(global::Android.Net.Uri contentUri)
         {
-            // if this is a file, use that
-            if (contentUri.Scheme == "file")
```

Comment:
I am not sure if I would drop this as it may have implications on older devices and APIs

Reply:
I've re-added this. Maybe I need to do a version branching thing here? Or is this fine?

Reply:
And yes, I just updated android to use the new permission api, it appeared to have gotten left behind xD. I removed the ensure check, because if I understood correctly RequestAsync already does an ensure check of its own.

Reply:
There shouldn't be any changes for the user

Reply:
I see the code is still missing? Have you pushed? Or am I misunderstanding this thread?

Reply:
The commit is below "Partially revert for older devices"